zebra-state/src/service/finalized_state/zebra_db/block/tests/snapshots/address_balances@mainnet_1.snap

```diff
@@ -4,7 +4,7 @@ expression: stored_address_balances
 ---
 [
   ("t3Vz22vK5z2LcKEdg16Yv4FFneEL1zg9ojd", AddressBalanceLocation(
-    balance: Amount(12500),
+    balance: 12500,
```

Comment:
I changed this to get `Amount` to serialize correctly in RPC fields.

That way, we don't have to convert to i64 first, the type just serializes automatically.

Reply:
Yes